.github/workflows/build-test.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   build-lint-test:
     name: Build, Lint, and Test
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Nit: We can also just use ubuntu-latest ?

Reply:
I see that in the module template we use `ubuntu-latest`, so that makes sense